replay/backend/managers/OverlayManager.ts

```diff
@@ -37,11 +37,11 @@ export default class OverlayManager {
   }
 
   public destroy = () => {
-    this.browserViews.forEach(x => x.destroy());
+    this.browserViews.length = 0;
   };
 
   public sendToAll = (channel: string, ...args: any[]) => {
-    this.browserViews.forEach(x => !x.isDestroyed() && x.webContents.send(channel, ...args));
+    this.browserViews.forEach(x => x?.webContents.send(channel, ...args));
```

Comment:
What prompted this change?

Reply:
Is it due to the electron upgrade?

Reply:
Electron got rid of these commands. They "say" it operates like a normal js object now.